tests/test_exacttarget_pagination.py

```diff
@@ -0,0 +1,52 @@
+from base import ExactTargetBase
+import tap_tester.connections as connections
+import tap_tester.menagerie as menagerie
+import tap_tester.runner as runner
+
+class ExactTargetPagination(ExactTargetBase):
+
+    def name(self):
+        return "tap_tester_exacttarget_pagination"
+
+    def get_properties(self, *args, **kwargs):
+        props = super().get_properties(*args, **kwargs)
+        props['batch_size'] = '1'
+        return props
+
+    def test_run(self):
+        page_size = 1
```

Comment:
If we have records then use at least 2 instead of one to test pagination properly.

Reply:
Email has 2 records only. So, we cannot use page size 2.